### PR TITLE
Makes Mosins bulky, lets you saw them into Obrezes [2]

### PR DESCRIPTION
**Pull request by Ryll-Ryll**
## About The Pull Request
I was dumb and made the last PR from my master branch instead of a branch branch, see #44774 for past discussion

Makes the Mosin Nagant rifle bulky instead of normal sized, but allows you to saw them off much like your average shotgun to normal size. However, sawing it down will introduce bullet spread because of the drastically reduced barrel.

Sprites:

Accuracy when sawed off (distance of 6 tiles, 4 hits out of 8 shots):

## Why It's Good For The Game
The Mosin Nagant is a really fucking big gun, 48.5 inches in its full length model, and with a bayonet attached it can reach near 6 feet long in total. Item size isn't 1:1 with real life, but considering just how notorious they are for being really big, paying homage to it feels right. Plus, running around with sawed off guns feel much more illegal and thus fun.

## Changelog
:cl: Shaps/Ryll
tweak: Mosin Nagants are now bulky, but can be sawed off to fit in a bag
/:cl:

### CODE REVIEW

**Comment by Shapsy** [comment]
Dropped the price on the Nuge Op version per @81Denton

**Comment by knacker48** [comment]
I am completely neutral about this change. I honestly never see Mosins being used except for when wizards summon guns

**Comment by HomocidlePancakes** [comment]
> I honestly never see Mosins being used except for when wizards summon guns

Every war op round.

File: code/modules/projectiles/guns/ballistic/rifle.dm [hunk 1]
```diff
@@ -59,6 +61,30 @@ obj/item/gun/ballistic/rifle/attackby(obj/item/A, mob/user, params)
 	knife_x_offset = 27
 	knife_y_offset = 13
 
+/obj/item/gun/ballistic/rifle/boltaction/attackby(obj/item/A, mob/user, params)
+	..()
+	if(istype(A, /obj/item/circular_saw) || istype(A, /obj/item/gun/energy/plasmacutter))
```
Comment (CPTANT) [inline code/modules/projectiles/guns/ballistic/rifle.dm]: Still a bad way to check for what items can cut a gun.

**Comment by CPTANT** [comment]
I saw cargo get them from a russian surplus crate? They are decent guns when there is nothing else.

File: code/modules/projectiles/guns/ballistic/rifle.dm (hunk 1 above)
Comment (Shapsy) [reply code/modules/projectiles/guns/ballistic/rifle.dm]: They're saws, that's what they do

**Pull request merged**